main/.vitepress/config.mjs

```diff
@@ -130,17 +130,20 @@ export default defineConfig({
               items: [
                 {
                   text: 'Takeaway 1: Starting a Local Chain',
-                  link: '/guides/getting-started/explainer-how-to-start-a-local-chain',
+                  link:
+                    '/guides/getting-started/explainer-how-to-start-a-local-chain',
```

Comment:
Is this change due to your local prettier or repo's prettier? If later then ok, otherwise you can revert these line changes.